src/commands/classes/_common.ts

```diff
@@ -0,0 +1,89 @@
+import ExecutionContext from '../../ExecutionContext';
+import * as Discord from '../../Discord';
+
+export type dbClass = {
+  name: string;
```

Comment:
Add the class CRN to the structure

```suggestion
  crn: string;
  name: string;
```